modules/apps/site/site-api/src/main/java/com/liferay/site/display/context/GroupDisplayContextHelper.java

```diff
@@ -25,6 +31,15 @@ public GroupDisplayContextHelper(HttpServletRequest httpServletRequest) {
 		_httpServletRequest = httpServletRequest;
 	}
 
+	public List<BreadcrumbEntry> getBreadcrumbEntries() {
+		BreadcrumbEntry breadcrumbEntry = new BreadcrumbEntry();
+
+		breadcrumbEntry.setTitle(
+			LanguageUtil.get(LocaleUtil.getDefault(), "staging"));
```

Comment:
Unless there's no choice, you shouldn't use the default locale. You have the request in scope, so get the current locale out of there (e.g. through the theme display).